compiled_starters/haskell/package.yaml

```diff
@@ -15,12 +15,19 @@ dependencies:
 - network-simple  # establish a simple TCP network
 - bytestring  # useful to send bytes over the network
 - network  # required when using bytestring
-- megaparsec  # may be useful for parsing the input
-- parser-combinators  # contains a helpful count function
 - text  # use Text rather than String
 - stm  # may help with a concurrent database
 - containers  # may help with the database structure and debugging
 - time  # may help with the expiry function
+# for overengineering
```

Comment:
@sherubthakur would remove these comments - just keep the dependencies, and explain what they can be used for